docs/get-started.rst

```diff
@@ -182,6 +182,11 @@ Make sure you have the system requirements listed in the
     cd kinto/
     make serve
 
+At the end of the installation, you will be asked which backend you would like to use:
```

Comment:
With restructured text, the way to have a code block is by either using `::` or by using the `code-block` directive, like this:


```rst

.. code-block:: bash

   some code here
```